tinyagentos/projects/storybook.py

```
@@ -0,0 +1,178 @@
+    return new.crop((left, top, left + box_w, top + box_h))
+
+
+def _open(image: Path | str | None) -> Image.Image | None:
+    if not image:
+        return None
+    try:
+        p = Path(image)
+        if not p.is_file():
+            return None
+        return Image.open(p).convert("RGB")
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)